libs/designer-ui/src/lib/unitTesting/outputMocks/outputMocks.tsx

```diff
@@ -59,7 +48,36 @@ export interface OutputsField extends SettingProps {
   onValueChange?: ChangeHandler;
 }
 
-export const OutputMocks = ({ isMockSupported, nodeId, onActionResultUpdate, outputs, mocks }: OutputMocksProps) => {
+export interface OutputMocksProps {
+  isMockSupported: boolean;
+  nodeId: string;
+  onActionResultUpdate: ActionResultUpdateHandler;
+  outputs: OutputsField[];
+  mocks: OutputMock;
+  errorMessage: string;
+  onErrorMessageChange: (value: string) => void;
+  errorCode: string;
+  onErrorCodeChange: (value: string) => void;
+  onMockUpdate: any;
+}
+
+export const ActionResults = {
```

Comment:
NIT: You don't need to re-declare the ActionResults here. You can get those from `@microsoft/designer-ui`
You can try something like thsi: ` import { ActionResults } from '@microsoft/designer-ui';`

Reply:
+1, I changed ActionResults in my PR to remove Skipped and Timedout so its better for this to inherit from that